examples/grails-database-per-tenant/build.gradle

```diff
@@ -15,7 +14,6 @@ dependencies {
     implementation 'org.grails:grails-plugin-rest'
     implementation 'org.grails.plugins:gsp'
 
-    runtimeOnly 'com.bertramlabs.plugins:asset-pipeline-grails'
```

Comment:
This one should be kept, as there are assets in the app. It's only the gradle plugin that can be removed. The gradle plugin will pre-compile assets when packaging the app. We won't do that with a test app. However the library is still needed at runtime when running tests to serve assets.